collect_app/src/main/java/org/odk/collect/android/forms/FormsRepository.java

```diff
@@ -12,6 +12,8 @@
 
     List<Form> getAll();
 
+    List<Form> getAll(String jrFormId, @Nullable String jrVersion);
```

Comment:
Hmmm I think we probably want to be more consistent here and make this a `getBy...` that returns a list. Maybe the methods that return a single item should be `getOne...` or `findOne...`?

Reply:
`findByXYZ` and `findOneXYZ`.

Reply:
Thanks, this was a big improvement.